DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -193,8 +193,8 @@
     <![CDATA[superClusterRelinked_.reset();]]>
   </ioread>
 
-  <class name="pat::Jet"  ClassVersion="14">
-   <version ClassVersion="15" checksum="1826981639"/>
+  <class name="pat::Jet"  ClassVersion="15">
+   <version ClassVersion="15" checksum="727883729"/>
```

Comment:
is this a conflict in version numbers?
If so, it should probably be resolved/merged in a different way.